cli/Cargo.toml

```diff
@@ -27,6 +28,11 @@ clap = { workspace = true, features = ["derive"] }
 serde = { workspace = true, features = ["derive"] }
 directories.workspace = true
 
+topiary = { workspace = true, optional = true}
+tree-sitter-nickel = { workspace = true, optional = true }
+tempfile.workspace = true
+thiserror.workspace = true
```

Comment:
Why do we need to pull those dependencies non-optionally?

Reply:
Good catch. In fact, `thiserror` isn't used at all anymore and I've made `tempfile` optional.